sphinx_needs/builder.py

```diff
@@ -158,3 +156,72 @@ def build_needumls_pumls(app: Sphinx, _exception: Exception) -> None:
         needs_builder.set_environment(env)
 
     needs_builder.finish()
+
+
+class NeedsIdBuilder(Builder):
```

Comment:
Do we need a separate builder for this or can we combine this in the current NeedsBuilder and differentiate using the config `needs_build_json_per_id`. Both builders from the user perspective is really similar and the only difference is how the output is sorted and stored. What do you say?

Reply:
I think we need ideas from @danwos  and @r-o-b-e-r-t-o

Reply:
For me it should be a different builder, because you maybe need to use both and you don't want to change the config everytime.
I see no drawback in having another builder for it.